helios-client/src/main/java/com/spotify/helios/client/HeliosClient.java

```diff
@@ -502,10 +505,22 @@ public static Builder newBuilder() {
 
   public static class Builder {
 
+    private static final String HELIOS_CERT_PATH = "HELIOS_CERT_PATH";
+
     private String user;
+    private Path clientCertificatePath;
+    private Path clientKeyPath;
     private Supplier<List<Endpoint>> endpointSupplier;
     private boolean sslHostnameVerification = true;
 
+    private Builder() {
+      final String heliosCertPath = System.getenv(HELIOS_CERT_PATH);
+      if (!isNullOrEmpty(heliosCertPath)) {
+        this.clientCertificatePath = Paths.get(heliosCertPath, "cert.pem");
+        this.clientKeyPath = Paths.get(heliosCertPath, "key.pem");
```

Comment:
are these standard names for these files? You could also just set one env var for each path.